static/js/pages/certificates/BrowserImport.tsx

```diff
@@ -0,0 +1,76 @@
+import React, { FC, useState } from "react";
+import { Col, Row, Tabs } from "@canonical/react-components";
+
+const TABS: string[] = ["Firefox", "Chrome"];
+
+const BrowserImport: FC = () => {
+  const [activeTab, handleTabChange] = useState("firefox");
+
+  return (
+    <Row>
+      <Col size={8}>
+        <Tabs
+          links={TABS.map((tab) => ({
+            label: tab,
+            active:
+              tab.toLowerCase() === activeTab ||
```

Comment:
Do we need to have it all lowercase for the `activeTab` state? Having it consistent with the tab title would save us a `toLowerCase` conversion.